evaluation-tasks/coughvid.py

```diff
@@ -90,11 +90,45 @@ def run(self):
             pass
 
 
+class FilterLabeledMetadata(WorkTask):
+    """
+    Filter the metadata (labels) to only contain audiofiles that
+    are labeled, and save it in metadata.csv with columns:
+    filename (without extension), label
+    """
```

Comment:
This is great -- can see how this could be extended / generalized to support different types of metadata filtering. Will be super helpful.